src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div>
+          <label class="form_field">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            Name:
+            <input
+              type="text"
+              name="name"
+              maxlength="20"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            How old are You?
+            <input
+              type="number"
+              name="number"
```

Comment:
Change the name attribute value of the age input to 'age' instead of 'number' for better semantics.